### PR TITLE
Anomali ThreatStream update

### PR DESCRIPTION
**Pull request by dantavori**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6323

# Screenshots
there are issues with the instance but the change looks ok:
Screenshot 2023-05-23 at 20 37 47
Screenshot 2023-05-23 at 20 37 59
then, when getting the import details:
image
Screenshot 2023-05-23 at 20 38 13

### CODE REVIEW

File: Packs/Anomali_ThreatStream/TestPlaybooks/playbook-ThreatStream-Test.yml [hunk 1]
```diff
@@ -735,7 +737,8 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-    continueonerrortype: ""
+    continueonerror: true
```
Comment (ShahafBenYakir) [inline Packs/Anomali_ThreatStream/TestPlaybooks/playbook-ThreatStream-Test.yml:740]: porque?

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py [hunk 2]
```diff
@@ -2070,16 +2070,17 @@ def get_passive_dns(client: Client, value, all_results=None, type=DBotScoreType.
...
     """
         Imports indicators data to ThreatStream.
         The data can be imported using one of three import_types: data-text (plain-text),
         file-id of uploaded file to war room or URL.
     """
-    # prepare
     data = assign_params(
         classification=classification,
         confidence=int(confidence),
+        source_confidence_weight=int(source_confidence_weight) if source_confidence_weight else None,
```
Comment (ShahafBenYakir) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py:2083]:
```suggestion
        source_confidence_weight=arg_to_number(source_confidence_weight) if source_confidence_weight else None,
```

**Review by ShahafBenYakir — APPROVED** [review]
Need to update the integration README

File: Packs/Anomali_ThreatStream/TestPlaybooks/playbook-ThreatStream-Test.yml (hunk 1 above)
Comment (dantavori) [reply Packs/Anomali_ThreatStream/TestPlaybooks/playbook-ThreatStream-Test.yml:740]: there's an issue with this command (not the one I modified, a different one) on the 3rd party product side (happens also in their UI)
so I continue on error, and skip the condition task that validates the command in the TPB until they resolve it...
@bakatzir FYI

Comment (ShahafBenYakir) [reply Packs/Anomali_ThreatStream/TestPlaybooks/playbook-ThreatStream-Test.yml:740]: hmm are we tracking it an any way? maybe we can open an internal ticket?

Comment (dantavori) [reply Packs/Anomali_ThreatStream/TestPlaybooks/playbook-ThreatStream-Test.yml:740]: I will open a ticket

**Pull request merged**